.github/workflows/examples.yml

```diff
@@ -72,20 +61,16 @@ jobs:
           restore-keys: |
             ${{ runner.os }}-pnpm-store-${{ matrix.dir }}-
 
-      - name: Cache Playwright's binary
-        uses: actions/cache@v3
-        with:
-          key: ${{ runner.os }}-playwright-bin-v1.26.0
-          path: ${{ env.PLAYWRIGHT_BROWSERS_PATH }}
-
       - run: node -v
       - name: Install deps and build (with cache)
         run: pnpm --filter ./examples/${{ matrix.dir }}  --filter "@trpc/*" --filter root install
 
+      - run: echo ${{ github.workspace }}/${{ matrix.dir }}/.next/cache
```

Comment:
the nextjs cache is never hitting, any ideas @sachinraja. I added this to try see if the path was wrong but seems to be good 🤔